app/src/main/res/layout-land/fragment_onboarding.xml

```diff
@@ -78,4 +79,183 @@
         app:layout_constraintTop_toTopOf="parent"
         />
   </androidx.constraintlayout.widget.ConstraintLayout>
+
+  <androidx.cardview.widget.CardView
+      android:id="@+id/onboarding_recover_guest_wallet"
+      android:layout_width="match_parent"
+      android:layout_height="76dp"
+      android:layout_marginHorizontal="16dp"
+      android:layout_marginBottom="16dp"
+      android:backgroundTint="@color/styleguide_blue_secondary"
+      android:minHeight="200dp"
+      android:visibility="invisible"
+      app:cardCornerRadius="16dp"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      tools:visibility="visible"
+      >
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        >
+
+      <TextView
+          android:id="@+id/onboarding_recover_text1"
+          style="@style/OnboardingValuesBodyTextStyle"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="16dp"
+          android:gravity="center"
+          android:text="@string/intro_from_web_payment_already_have_title"
+          app:layout_constraintBottom_toBottomOf="@+id/onboarding_recover_guest_button"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="@+id/onboarding_recover_guest_button"
+          />
+
+      <com.google.android.material.card.MaterialCardView
+          android:id="@+id/onboarding_bonus_layout"
+          android:layout_width="wrap_content"
+          android:layout_height="38dp"
```

Comment:
prefer to use dynamic sizes i

Reply:
yes it would be better, but the animation item is bigger than the parent

Reply:
it is not a design guideline, but ok, I will accept it